control-plane/api/v1alpha1/ingressgateway_types_test.go

```diff
@@ -50,6 +50,9 @@ func TestIngressGateway_MatchesConsul(t *testing.T) {
 							ClusterName:  "cluster1",
 							CertResource: "cert1",
 						},
+						TLSMinVersion: "TLSv1_0",
+						TLSMaxVersion: "TLSv1_1",
+						CipherSuites:  []string{"ECDHE-ECDSA-AES128-GCM-SHA256", "AES128-SHA"},
```

Comment:
Do we need any specific tests for CipherSuites?

Reply:
The list of CipherSuites is super expansive and is not static. We will rely on Consul to fail to sync in case an invalid cypher suite is used.